test/integration/targets/special_vars/tasks/main.yml

```diff
@@ -56,3 +56,12 @@
   assert:
     that:
       - "(ansible_play_role_names + ansible_dependent_role_names)|unique|sort|list == ansible_role_names|sort|list"
+
+- name: test ansible_parent_role_name(s) functionality by including another role
+  include_role:
```

Comment:
We need tests for `import_role` as well.

Reply:
Done.